infra/modules/providers/azure/app-service/main.tf

```diff
@@ -39,6 +39,7 @@ resource "azurerm_app_service" "appsvc" {
     linux_fx_version     = format("DOCKER|%s/%s", var.docker_registry_server_url, local.app_configs[count.index].image)
     always_on            = var.site_config_always_on
     virtual_network_name = var.vnet_name
+    ip_restriction       = []
```

Comment:
FYI, this is done to resolve a spurious error during what should be a clean subsequent APPLY (when no changes were made to schema).